docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -305,10 +305,11 @@ paths:
           description: The field by which annotations should be sorted.
           required: false
           type: string
+          enum: [created, updated, group, id, tags_raw, raw_user, uri]
           default: updated
         - name: order
           in: query
-          description: The order in which the results should be sorted.
+          description: The direction of sort.
```

Comment:
I felt like this description was more concise and hinted at the fact that there are specific directions but if you don't like it I can change it back.

Reply:
I found the previous description marginally clearer myself.

Reply:
Changed it back.